src/node/entities.h

```diff
@@ -60,50 +60,68 @@ namespace ccf
 
   struct Tables
   {
-    // Governance tables
+    // Service tables
+
+    // Member identities
     static constexpr auto MEMBERS = "public:ccf.gov.members";
-    static constexpr auto MEMBER_ACKS = "public:ccf.gov.member_acks";
-    static constexpr auto MEMBER_CERT_DERS = "public:ccf.gov.member_cert_ders";
-    static constexpr auto MEMBER_DIGESTS = "public:ccf.gov.member_digests";
+    static constexpr auto MEMBER_ACKS = "public:ccf.gov.members.acks";
+    static constexpr auto MEMBER_CERT_DERS =
+      "public:ccf.internal.members.certs_der";
+    static constexpr auto MEMBER_DIGESTS =
+      "public:ccf.internal.members.digests";
+
+    // User identities
     static constexpr auto USERS = "public:ccf.gov.users";
-    static constexpr auto USER_CERT_DERS = "public:ccf.gov.user_cert_ders";
-    static constexpr auto USER_DIGESTS = "public:ccf.gov.user_digests";
+    static constexpr auto USER_CERT_DERS =
+      "public:ccf.internal.users.certs_der";
+    static constexpr auto USER_DIGESTS = "public:ccf.internal.users.digests";
     static constexpr auto SERVICE_PRINCIPALS =
       "public:ccf.gov.service_principals";
+
+    // Nodes identities and allowed code ids
     static constexpr auto NODES = "public:ccf.gov.nodes";
-    static constexpr auto VALUES = "public:ccf.gov.values";
-    static constexpr auto CONSENSUS = "public:ccf.gov.consensus";
-    static constexpr auto WHITELISTS = "public:ccf.gov.whitelists";
+    static constexpr auto NODE_CODE_IDS = "public:ccf.gov.nodes.code_ids";
+
+    // Service information
+    static constexpr auto SERVICE = "public:ccf.gov.service";
+    static constexpr auto CONFIGURATION = "public:ccf.gov.service.config";
+
+    // Governance
     static constexpr auto PROPOSALS = "public:ccf.gov.proposals";
     static constexpr auto GOV_SCRIPTS = "public:ccf.gov.governance.scripts";
-    static constexpr auto APP_SCRIPTS = "public:ccf.gov.app_scripts";
-    static constexpr auto MODULES = "public:ccf.gov.modules";
-    static constexpr auto SECRETS = "public:ccf.gov.secrets";
-    static constexpr auto NODE_CODE_IDS = "public:ccf.gov.nodes.code_ids";
     static constexpr auto GOV_HISTORY = "public:ccf.gov.governance.history";
-    static constexpr auto SERVICE = "public:ccf.gov.service";
-    static constexpr auto SHARES = "public:ccf.gov.shares";
-    static constexpr auto CONFIGURATION = "public:ccf.gov.config";
-    static constexpr auto SUBMITTED_SHARES = "public:ccf.gov.submitted_shares";
-    static constexpr auto SNAPSHOT_EVIDENCE =
-      "public:ccf.gov.snapshot_evidence";
-    static constexpr auto CA_CERT_DERS = "public:ccf.gov.ca_cert_ders";
-    static constexpr auto JWT_ISSUERS = "public:ccf.gov.jwt_issuers";
+    static constexpr auto WHITELISTS = "public:ccf.gov.whitelists";
```

Comment:
Is this member writable? It also needs renaming.

Reply:
I believe it is. It is set to a default value (see `default_whitelists.h`) but I'm not sure we have any end-to-end test that modify this table. I'm assuming that this will change/disappear once we have JS governance?

Reply:
Yes